From 082d2eb2a65525890a913723764e67a36ee75384 Mon Sep 17 00:00:00 2001
From: Scott Murray <scott.murray@konsulko.com>
Date: Fri, 13 Sep 2019 19:26:27 -0400
Subject: [PATCH] Include signal.h

Fixes several signal set related errors:
src/basic/copy.c:92:19: error: implicit declaration of function 'sigemptyset' [-Werror=implicit-function-declaration]
src/basic/copy.c:93:19: error: implicit declaration of function 'sigaddset' [-Werror=implicit-function-declaration]
src/basic/copy.c:93:34: error: 'SIGINT' undeclared (first use in this function)
src/basic/copy.c:95:13: error: implicit declaration of function 'sigtimedwait' [-Werror=implicit-function-declaration]

Upstream-Status: Pending

Signed-off-by: Scott Murray <scott.murray@konsulko.com>

---
 src/basic/copy.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/basic/copy.c b/src/basic/copy.c
index 9028868f696d..5168586fa522 100644
--- a/src/basic/copy.c
+++ b/src/basic/copy.c
@@ -8,6 +8,7 @@
 #include <sys/sendfile.h>
 #include <sys/xattr.h>
 #include <unistd.h>
+#include <signal.h>
 
 #include "alloc-util.h"
 #include "btrfs-util.h"
